blockly-core/core/ui/block_space/trashcan.js

```diff
@@ -86,7 +86,7 @@ Blockly.Trashcan.prototype.MARGIN_SIDE_ = 22;
 * @type {number}
 * @private
 */
-Blockly.Trashcan.prototype.MARGIN_HOTSPOT_ = 25;
+Blockly.Trashcan.MARGIN_HOTSPOT_ = 25;
```

Comment:
Good point—for ones that aren't referenced externally it probably makes sense switch over to all-caps local variables with `@const` set per the first example in: https://developers.google.com/closure/compiler/docs/js-for-compiler